### PR TITLE
css two column layout, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    </div>
+    <div class="row">
+      <div class="column-full wtf">
+        <p>We've dedicated thousands of man hours to making healthcare
+          institutions in the Helium network run 10x more efficiently and
+          deliver an unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-sixth">
```
Comment (Zhiwutian) [inline css-two-column-layout/index.html:27]: only thing I would recommend for the HTML is to wrap the `column-one-sixth` and `column-two-sixths` elements with a `column-one-half` as the parent so you have two total columns per row:
```html
<div class="column-half">
   <div class="column-one-sixth">
    <img src="images/circumstance.svg" alt="Circumstance">
  </div>
  <div class="column-two-sixths space">
    <h3>FOR ANY CIRCUMSTANCE</h3>
    <p>Our intuitive system is built around the
    peculiarities of your facility; we support
    offline deployments, sync to the cloud
    and run on any device 24/7.</p>
  </div>
</div>
```

Keep up the good work!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small HTML update, looking good!

**Comment by avallonsmith** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**